spec/features/user_spec.rb

```diff
@@ -8,23 +8,25 @@
 feature 'OpenSourceCMS test', type: :feature do
 
   describe Pages::Users do
```

Comment:
It is better to use the following format for describes 
describe 'When on User Page'
it 'verifies user can be created' do
...
end

it 'verifies user can be edited' do
...
end
 
to test the output, you can run tests with --format documentation parameter